platform/security/idp/security-idp-plugin/security-idp-plugin-namequalifier/src/test/groovy/org/codice/ddf/security/idp/plugin/namequalifier/NameQualifierPresignPluginSpec.groovy

```diff
@@ -173,6 +199,25 @@ class NameQualifierPresignPluginSpec extends Specification {
 
     }
 
+    def 'test name qualifier when assertion subject is not null and name id is not persistent'() {
+
+        setup:
+        response.getIssuer() >> issuer
+        response.getIssuer().getFormat() >> format
+        assertions.add(assertion)
+        assertion.getSubject() >> subject
+        assertion.getSubject().getNameID() >> nameIDNeg
+        nameIDNeg.getFormat() >> NEGATIVE_TEST
+
+        when:
+        plugin.processPresign(response, authNReq, spMetadata, bindings)
+
+        then:
+        0 * nameID.getFormat()
+
+
```

Comment:
✏️ This test and the one above it have two empty lines at the end. They should only have a max of one.